Javascript ④－1/js/main.js

```diff
@@ -20,13 +20,12 @@
 
     class Quiz {
 
-      constructor(text1, text2, text3, text4, text5, text6) {
+      constructor(text1, text2, text3, text4, text5,) {
```

Comment:
この数だと確かに命名は面倒なのですが…
やはりこのように連番で付けてしまうと第三者（**未来の自分含む**）からすると
どこに何を設定すればよいかがすぐに分からないので、見るのがつらいコードになってしまいます。

あと、削除漏れと思われるカンマが残っています。
```suggestion
      constructor(text1, text2, text3, text4, text5, {
```